src/deploy/functions/params.ts

```diff
@@ -334,11 +343,13 @@ export async function resolveParams(
 
 function populateDefaultParams(config: FirebaseConfig): Record<string, ParamValue> {
   const defaultParams: Record<string, ParamValue> = {};
-  defaultParams["DATABASE_URL"] = new ParamValue(config.databaseURL, true, {
-    string: true,
-    boolean: false,
-    number: false,
-  });
+  if (config.databaseURL !== "") {
+    defaultParams["DATABASE_URL"] = new ParamValue(config.databaseURL, true, {
+      string: true,
+      boolean: false,
+      number: false,
+    });
+  }
   defaultParams["PROJECT_ID"] = new ParamValue(config.projectId, true, {
```

Comment:
nit* I wonder if we need PROJECT_ID. We reserve GCLOUD_PROJECT and not PROJECT_ID - what would happen if user had defined `PROJECT_ID` in the dotenv file 🤔

Reply:
The user's definition wins. I added a test case to make this clear.